marlowe-playground-client/src/BottomPanel/Types.purs

```diff
@@ -0,0 +1,44 @@
+module BottomPanel.Types where
+
+import Prelude
+import Analytics (class IsEvent, Event)
+import Analytics as A
+import Data.Lens (Lens')
+import Data.Lens.Record (prop)
+import Data.Maybe (Maybe(..))
+import Data.Symbol (SProxy(..))
+
+-- This action is parameterized in the `panel` type so we can have type safety on the different panels
+-- we have, and the panelAction, which is needed to allow the contents of the panel, to fire actions
+-- on the parent. The `panel` type could eventually be refactored into just having a Map/Record, if we dont
+-- care for it to be type safe, and the PanelAction could eventually be refactored to Variants/Run.
+data Action panel panelAction
```

Comment:
could you explain a bit more about what a `panel` is in the specific context of this UI element. I.e. "the bottom panel has different tabs, panel defines which tab is selected" or something

Reply:
renamed panelAction and added better explanation